samples/wifi/raw_tx_packet/README.rst

```diff
@@ -0,0 +1,166 @@
+.. _wifi_raw_tx_packet_sample:
+
+Wi-Fi: Raw TX packet
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Raw TX packet sample demonstrates how to transmit raw TX packets.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample demonstrates how to transmit raw TX packets in both connected Station and non-connected Station modes of operation.
+The sample provides the option to select the either of the modes of transmission:
+
+* Continuous packet transmission.
+* Fixed number of packet transmission.
+
+The configurations for connected Station or non-connected Station modes, and for continuous or fixed packet transmission, are set at build time.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+The following sample-specific Kconfig options are used in this sample (located in :file:`samples/wifi/raw_tx_packet/Kconfig`):
+
+.. options-from-kconfig::
+
+Configuration options for connected Station mode
```

Comment:
Could we add some indentation here, for ease of reading? That is to ensure that the reader understand what configuration is needed in each case:

- Operating mode
  - Configuration options for connected Station mode
  - Configuration options for non-connected Station mode
- Configuration options for Raw TX packet header
- Misc (or, Additional configurations)